examples/beam/src/test/java/edu/snu/nemo/examples/beam/AlternatingLeastSquareITCase.java

```diff
@@ -33,19 +35,32 @@
   private static final int TIMEOUT = 180000;
   private static final String fileBasePath = System.getProperty("user.dir") + "/../resources/";
   private static final String input = fileBasePath + "sample_input_als";
+  private static final String outputFileName = "sample_output_als";
+  private static final String output = fileBasePath + outputFileName;
+  private static final String testResourceFileName = "test_output_als";
   private static final String numFeatures = "10";
   private static final String numIteration = "3";
+  // If you don't want to check validity of output, make this variable false.
+  private static final boolean checkOutput = true;
 
   private static ArgBuilder builder = new ArgBuilder()
       .addJobId(AlternatingLeastSquareITCase.class.getSimpleName())
       .addUserMain(AlternatingLeastSquare.class.getCanonicalName())
-      .addUserArgs(input, numFeatures, numIteration);
+      .addUserArgs(input, numFeatures, numIteration, "0.05", output);
 
   @Before
   public void setUp() throws Exception {
     builder = new ArgBuilder()
         .addUserMain(AlternatingLeastSquare.class.getCanonicalName())
-        .addUserArgs(input, numFeatures, numIteration);
+        .addUserArgs(input, numFeatures, numIteration, "0.05", output);
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    if (checkOutput) {
+      ExampleTestUtil.ensureALSOutputValidity(fileBasePath, outputFileName, testResourceFileName);
+      ExampleTestUtil.deleteOutputFile(fileBasePath, outputFileName);
+    }
   }
 
 //  @Test (timeout = TIMEOUT)
```

Comment:
Please remove unused code.

Reply:
I think this unused code is remained intentionally.

Reply:
Can you leave a comment here to explain why? It'd be nice to add a TODO as well, so that we can look up related issues quickly when we decide to work on it.

Reply:
I don't know why it is commented out and who has did it. I've just uncommented the test and it seems work nicely.

Reply:
I remember that we just commented out this test to reduce the IT time. It would be okay to remove, re-enable, or add comment for it.